Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -528,7 +530,7 @@ void MainWindow::load_plugin(QString fileName, bool blacklisted)
       {
         if ( plugin_blacklist.contains(name) ){
           qDebug("### Ignoring plugin \"%s\".", qPrintable(fileName));
-          return;
+          return true;
```

Comment:
I used this retur value to get all files that are plugins, not only those that are loaded, as the filtering is done in load_plugin() and not in load_plugins().